rollup.config.dist.js

```diff
@@ -10,7 +10,11 @@ var banner = `/**
  */
 `;
 
-export default {
+var typeref = `/// <reference types="https://cdn.jsdelivr.net/gh/fluture-js/Fluture@${
+  process.env.VERSION || pkg.version
+}/index.d.ts" />`;
```

Comment:
This is what's supposed to tell TypeScript compilers where to look for types. However, in my tests it does not appear to be working. See https://deno.land/manual/getting_started/typescript#triple-slash-reference-directive-in-javascript-files for the docs I used.

Reply:
I found out why it's not working. It's because of the module declaration in my TypeScript typings saying that the typings only apply to the `"fluture"` import, whereas they should be applying to the `"https://cdn.jsdelivr.net/gh/fluture-js/Fluture@dist/dist/module.js"` import.

Reply:
This was fixed by removing the module declaration.